From dcd908c7f41df104388bf94ff6101ee2f7cfd1f5 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Mon, 17 Nov 2025 11:44:25 +0100
Subject: [PATCH] arm64: configs: reduce NR_CPUS from arm64 default 256 to 2

STM32MP2x family counts up to 2 CPUs. Reducing NR_CPUS from 256 to 2
reduces the kernel footprint.

Change-Id: I255aea8bc3d30e24b0a96df734010d380e77f7f4
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/509700
Reviewed-by: Jerome BESNARD <jerome.besnard@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index c5b1e68dc302..8202fe03fdd1 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -49,6 +49,7 @@ CONFIG_NET_CLS_U32=m
 CONFIG_NET_SCH_MULTIQ=m
 CONFIG_NET_SCH_PRIO=m
 CONFIG_NET_SCH_SKBPRIO=m
+CONFIG_NR_CPUS=2
 CONFIG_NVMEM_REBOOT_MODE=y
 CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_NVMEM_STM32_TAMP=y
-- 
2.53.0

